packages/rendermime/style/base.css

```diff
@@ -413,7 +413,7 @@
   border-spacing: 0;
   border: none;
   color: var(--jp-ui-font-color1);
-  font-size: 12px;
+  font-size: var(--jp-ui-font-size1);
```

Comment:
```suggestion
  font-size: calc(12/13 * var(--jp-ui-font-size1));
```

Reply:
the main question is do we _want_ to have table be 12/13 of the normal font size just for the sake of keeping a value that was chosen 6 years ago ? 

I haven't been involved in JupyterLab design in a long time, so I don't want my opinion to weight too much on this, but I believe that uniformity is better, and would personally avoid calc even if it make a slight change of theme